lib/backpex/html/resource/form_component.html.heex

```diff
@@ -53,7 +53,19 @@
           </button>
         </.link>
         <button
+          :if={@continue_label}
           type="submit"
+          name="save-type"
+          value="continue"
+          class={["btn", if(form_errors?(@show_form_errors, @form), do: "btn-error", else: "btn-primary")]}
```

Comment:
Maybe this should be `btn-neutral btn-outline` to not have two primary buttons at the same time? The primary button should indicate "enter" press.